packages/node_modules/@ciscospark/react-component-activity-menu/src/index.js

```diff
@@ -28,38 +29,33 @@ export default function ActivityMenu(props) {
     onMessage();
   }
 
+  const buttons = [
```

Comment:
I was thinking we could even pass this array as a prop to the menu, that way the top level would control which menu items are shown.